### ISSUE TITLE
Erroneous code comments on complex function signatures

### ISSUE THREAD

**Issue by lord-saumagen** [kasecato/vscode-docomment#88]
# The program creates erroneous code comments on complex function signatures.  

![Error_Report_2020-05-19](https://user-images.githubusercontent.com/9468599/82308744-81e20580-99c2-11ea-921e-6ebfc232a0e7.gif)

**Comment by kasecato** [comment]
```csharp
    public static List<object> ComplexFunctionsSignatureOne(List<object> someList,
                                                            Func<object, bool> delegateOne = null,
                                                            Func<object, bool> delegateTwo = null) { }

     public static (object result, string name) ComplexFunctionSignatureTwo() { }
```

**Comment by kasecato** [comment]
Hello @lord-saumagen , thank you for your reporting me about the issues. I fixed, please check it on [v0.1.11](https://marketplace.visualstudio.com/items?itemName=k--kato.docomment). Thank you

**Comment by lord-saumagen** [comment]
Wow, that was fast!

Thank you very much for you effort and of  cause for allowing me to user
your program.

Kind regards, Lord Saumagen.
